.gitpod.yml

```diff
@@ -117,4 +117,6 @@ vscode:
     - 'redhat.vscode-yaml'
     - 'davidanson.vscode-markdownlint'
     - 'streetsidesoftware.code-spell-checker'
+    - 'dejmedus.tailwind-sorter'
```

Comment:
Seems to be doing the same as the prettier sorting?

Reply:
Yes, that's the idea?

Reply:
I am auto formatting with prettier on save which would now already sort the classes. Isn't using that plugin redundant?

Reply:
Yeah they might be indeed doing the same. The vscode one can use custom sorting but if we don't need that, we can stick with the prettier plugin.

<img width="254" alt="image" src="https://github.com/user-attachments/assets/d032d042-3e86-42b9-85ab-ca6190d4890e" />

Reply:
I dropped it already.